packages/core/src/components/TextInput.tsx

```diff
@@ -1,25 +0,0 @@
-import React from "react";
```

Comment:
why are we deleting this file?

Reply:
Since we're removing `initialValue`, there's no longer a need for a wrapper component. We can just use the react-native TextInput